editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -60,6 +60,8 @@
 #include "scene/main/window.h"
 #include "scene/resources/packed_scene.h"
 #include "scene/resources/style_box_texture.h"
+#include <core/error/error_macros.h>
+#include <core/io/resource_loader.h>
```

Comment:
These are already included indirectly.

Reply:
Also for future PRs, such includes should use `""`, not `<>`. The latter is reserved for system headers.

Reply:
Sure, didn't notice as they got added by mi text editor; will change.

Reply:
Rebased and removed includes.